rosdep/python.yaml

```diff
@@ -2421,7 +2421,7 @@ python-influxdb:
     bionic: [python-influxdb]
     disco: [python-influxdb]
     xenial: [python-influxdb]
-python-inject-pip:
+python-inject-pip: &migrate_eol_2030_04_30_python3_inject_pip
```

Comment:
I think the date we use is April 2025. The extra 5 years is "Extended Security Maintenance" which requires an "Ubuntu Advantage Subscription"

https://ubuntu.com/about/release-cycle

Reply:
Okay, I've updated the date to 2025. Thanks